vkapi/vkapi.py

```diff
@@ -587,6 +587,11 @@ def __init__(self, registry, en):
     self.requires = [x for x in en.get('requires', '').split(',') if x]
     self.specialuse = [x for x in en.get('specialuse', '').split(',') if x]
 
+    # Read extension name and spec version enums.
+    enums = [t.get('name') for t in en.findall('require/enum[@value]')]
+    self.spec_version_enum = enums[0]
+    self.name_enum = enums[1]
```

Comment:
I'm a bit nervous about hard coding that the first 2 enums are `*_EXTENSION_NAME` and `*_SPEC_VERSION`. Can we do this based on the name?

Reply:
Done.